Microsoft.Toolkit.HighPerformance/Microsoft.Toolkit.HighPerformance.csproj

```diff
@@ -24,17 +24,19 @@
   </Description>
```

Comment:
@Sergio0694 mind reviewing these csproj file changes to the high performance package?

Reply:
The logic is the same as before, only rearranged some lines to improve readability.